exchanges/btse/btse_types.go

```diff
@@ -2,9 +2,24 @@ package btse
 
 import "time"
 
+// OverviewData stores market overview data
+type OverviewData struct {
+	High24Hr         float64 `json:"high24hr,string"`
+	HighestBid       float64 `json:"highestbid,string"`
+	Last             float64 `json:"last,string"`
+	Low24Hr          float64 `json:"low24hr,string"`
+	LowestAsk        float64 `json:"lowest_ask,string"`
+	PercentageChange float64 `json:"percentage_change,string"`
+	Volume           float64 `json:"volume,string"`
+}
+
+// HighLevelMarketData stores market overview data
+type HighLevelMarketData map[string]OverviewData
+
 // Market stores market data
 type Market struct {
 	Symbol              string  `json:"symbol"`
+	ID                  string  `json:"id"`
```

Comment:
This is never actually returned! 🦀 https://api.btse.com/spot/v2/markets
Keep it here anyway, everything still works